public/docs/ts/latest/guide/npm-packages.jade

```diff
@@ -150,16 +150,16 @@ a(id="dev-dependencies")
   They do not have to be deployed with the production application although there is rarely harm in doing so.
 
   ***[concurrently](https://www.npmjs.com/package/concurrently)*** - 
-  a utility to run multiple *npm* commands concurrently on OS/X, Windows, and Linux operating systems.
+  A utility to run multiple *npm* commands concurrently on OS/X, Windows, and Linux operating systems.
```

Comment:
Interesting. The weight of the writer world is in _your favor_.
I read several posts like [this one](https://owl.english.purdue.edu/owl/resource/591/01/) which say

>The choice of article is actually based upon the phonetic (sound) quality of the first letter in a word, not on the orthographic (written) representation of the letter. If the first letter makes a vowel-type sound, you use "an"; if the first letter would make a consonant-type sound, you use "a." 

News to me and English is my first language. 

Thanks.